src/rich-text-editor.component.js

```diff
@@ -226,16 +232,6 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
     divRef.current.focus()
     setFocused(true)
   }
-
-  function getBrowserDefaultStyling() {
-    if (props.placeholderColor) {
-      return `color: ${props.placeholderColor};`
-    } else if (!!window.chrome && (!!window.chrome.webstore || !!window.chrome.runtime)) {
-      return `color: rgb(117, 117, 117);`
-    } else {
-      return `opacity: 0.54;`
-    }
-  }
```

Comment:
should default styling still be provided?

Reply:
Yeah, let's keep the default styling.